src/test/resources/testfiles/conf/database-conf.xml

```diff
@@ -2,50 +2,50 @@
 <beans>
 <bean id="testDbDataSource"
       class="org.apache.commons.dbcp.BasicDataSource"
-      destroy-method="close">
+      destroy-method="close" scope="singleton">
```

Comment:
Could you explain about the changes?

Reply:
The default Spring bean scope is singleton (i.e. if scope attribute is not explicitly set) per http://blog.marcnuri.com/spring-bean-scopes-guide-to-understand-the-different-bean-scopes/. Given that all other beans in the file had scope set, I added this line during Spring framework upgrade to make the scope explicit and align with its explicit setting for other beans in the same file.